src/components/content/order/ServiceList.tsx

```diff
@@ -281,9 +311,21 @@ function ServiceList(): JSX.Element {
         getServices();
     }
 
+    const handleOk = () => {
+        setIsModalOpen(false);
+    };
+
+    const handleCancel = () => {
+        setIsModalOpen(false);
+    };
+
     return (
         <div className={'services-content'}>
             {tip}
+
+            <Modal title='Endpoint Information' width={700} open={isModalOpen} onOk={handleOk} onCancel={handleCancel}>
```

Comment:
Would be good if we can move this to a new EndpointInformation component and just use it here.